src/main/resources/entity-types/users/simple_FOLIO_user_details.json5

```diff
@@ -0,0 +1,154 @@
+{
+  id: 'f2615ea6-450b-425d-804d-6a495afd9308',
+  name: 'simple_FOLIO_user_details',
```

Comment:
Can we give this a more descriptive name? At a glance, it's not at all obvious what the difference between this and `simple_user_details` based on the names

Reply:
@mweaver-ebsco , or this PR can we have as it is:
https://github.com/folio-org/mod-fqm-manager/pull/495. Since if we are removing duplicate column, then all the flags needs to be reconsidered for other ET's as well. And kathleen wants it before freeze. I can create another for addressing your thoughts.